docs/common/wibox.ldoc

```diff
@@ -37,6 +37,13 @@
 -- @tparam[opt=false] boolean ontop
 -- @propemits false false
 
+--- Below other windows and widgets.
```

Comment:
What does `and widgets` refer to? Wiboxes/drawins are not ordered relative to widgets, are they? I'd just leave that part out

Reply:
it would probably be better to say `and wiboxes` or `and drawins`, or, as you said, just leave that part out entirely. not sure which one would be better. my bad for getting the terminology wrong lol